src/features/movies/MoviePage/index.js

```diff
@@ -72,18 +73,20 @@ export const MoviePage = () => {
                 <Header>Cast</Header>
                 <ContentContainer>
                   {cast.map(({ profile_path, name, character, id }) => (
-                    <Container key={id} id={id}>
-                      <ProfileImage
-                        src={
-                          profile_path
-                            ? `${imageBaseUrl}/w342${profile_path}`
-                            : noPicture
-                        }
-                        alt={`image of ${name}`}
-                      ></ProfileImage>
-                      <Name>{name}</Name>
-                      <Character>{character}</Character>
-                    </Container>
+                    <StyledLink to={`/profile/${id}`}>
+                      <Container key={id} id={id}>
+                        <ProfileImage
+                          src={
+                            profile_path
+                              ? `${imageBaseUrl}/w342${profile_path}`
+                              : noPicture
+                          }
+                          alt={`image of ${name}`}
```

Comment:
Fajna sprawa :D Wydaję mi się że lepiej by było wpisać w `alt={" portrait of ${name}`}`. Bo `alt` ma być opisem obrazka, a nie informacją że on tam jest. Jest to trochę mylące ;D

Reply:
Dobra, ustawie tak wszystkim altom ktore mamy w naszej aplikacji a ktore odnosza sie do osoby :)